src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderServiceTest.java

```diff
@@ -590,4 +593,71 @@ void notifySnapshotReady() {
         .snapshotReady(
             eq(id), anyString(), eq(snapshot.getName()), eq("No snapshot specification found"));
   }
+
+  @Test
+  void testGetRequestGroupMembers() {
+    SnapshotAccessRequestMembersResponse expected = new SnapshotAccessRequestMembersResponse();
+    expected.members(new ArrayList<>());
+    UUID requestId = new SnapshotAccessRequestResponse().getId();
+    SnapshotAccessRequestModel requestModel =
+        SnapshotBuilderTestData.createAccessRequestModelApproved();
+    when(snapshotRequestDao.getById(requestId)).thenReturn(requestModel);
+    when(iamService.getGroupPolicyEmails(requestModel.samGroupName(), IamRole.MEMBER.toString()))
+        .thenReturn(new ArrayList<>());
+    assertThat(snapshotBuilderService.getGroupMembers(requestId), is(expected));
+  }
+
+  @Test
+  void testAddRequestGroupMember() {
+    SnapshotAccessRequestMembersResponse expected = new SnapshotAccessRequestMembersResponse();
+    String memberEmail = "user@gmail.com";
+    expected.members(new ArrayList<>(List.of(memberEmail)));
+    UUID requestId = new SnapshotAccessRequestResponse().getId();
+    SnapshotAccessRequestModel requestModel =
+        SnapshotBuilderTestData.createAccessRequestModelApproved();
+    when(snapshotRequestDao.getById(requestId)).thenReturn(requestModel);
+    when(iamService.addEmailToGroup(
+            requestModel.samGroupName(), IamRole.MEMBER.toString(), memberEmail))
+        .thenReturn(new ArrayList<>(List.of("user@gmail.com")));
+    assertThat(snapshotBuilderService.addGroupMember(requestId, memberEmail), is(expected));
+  }
+
+  @Test
+  void testAddRequestGroupMemberInvalidEmail() {
+    String badEmail = "badEmail";
+    UUID requestId = new SnapshotAccessRequestResponse().getId();
+    SnapshotAccessRequestModel requestModel =
+        SnapshotBuilderTestData.createAccessRequestModelApproved();
+    when(snapshotRequestDao.getById(requestId)).thenReturn(requestModel);
+    assertThrows(
+        bio.terra.app.controller.exception.ValidationException.class,
```

Comment:
Instead of a fully qualified name, can you use an `import` here intead?
```suggestion
       ValidationException.class,
```